src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java

```diff
@@ -109,8 +109,9 @@ public static ColumnVector toIntegersWithBase(ColumnView cv, int base,
       type.getTypeId().getNativeId()));
   }
 
-  public static ColumnVector fromIntegersWithBase(ColumnView cv, int base) {
-    return new ColumnVector(fromIntegersWithBase(cv.getNativeView(), base));
+  public static ColumnVector fromIntegersWithBase(ColumnView cv, int base,
```

Comment:
We need to maintain backwards compatibility. We need to keep a version that does not take the boolean parameter and defaults it to true. And while you are at it we should have some comments on this to explain what it is doing so the parameters are clear.